test/test_transforms_v2_refactored.py

```diff
@@ -232,7 +232,7 @@ def _check_transform_v1_compatibility(transform, input, *, rtol, atol):
     """If the transform defines the ``_v1_transform_cls`` attribute, checks if the transform has a public, static
     ``get_params`` method that is the v1 equivalent, the output is close to v1, is scriptable, and the scripted version
     can be called without error."""
-    if type(input) is not torch.Tensor or isinstance(input, PIL.Image.Image):
+    if not (type(input) is torch.Tensor or isinstance(input, PIL.Image.Image)):
```

Comment:
This was spotted by @NicolasHug. We only want to test the v1 compatibility for pure tensors and PIL images. Fixing this, lead to two more changes that I'll flag below.